pages/about.vue

```diff
@@ -0,0 +1,145 @@
+<template>
```

Comment:
Looks good Ruby but I noticed this is the old About page. Is it worth just using the one that is on witches.ed now as that is the one we will want to launch with anyway?

Reply:
Thank you. Looks great.